### PR TITLE
[NFC] Remove unnecessary constant

### PR DESCRIPTION
**Pull request by keith**

### CODE REVIEW

File: utils/build-script [hunk 1]
```diff
@@ -1168,7 +1165,7 @@ def main_preset():
                 "build-presets.ini")
         ]
 
-        user_presets_file = os.path.join(HOME, '.swift-build-presets')
+        user_presets_file = os.path.join(os.getenv("HOME", "/"), '.swift-build-presets')
```
Comment (xwu) [inline utils/build-script:1168]:
```suggestion
        user_presets_file = os.path.join(os.getenv("HOME", "/"),
                                         '.swift-build-presets')
```

**Comment by CodaFi** [comment]
@swift-ci python lint

**Comment by CodaFi** [comment]
@swift-ci smoke test and merge

**Comment by CodaFi** [comment]
@swift-ci smoke test macOS

**Comment by CodaFi** [comment]
⛵

**Comment by keith** [comment]
Thanks! 😅

**Pull request merged**